src/networking.c

```diff
@@ -131,6 +131,7 @@ client *createClient(connection *conn) {
     c->conn = conn;
     c->name = NULL;
     c->bufpos = 0;
+    c->buf_usable_size = zmalloc_usable_size(c)-sizeof(*c)+sizeof(c->buf);
```

Comment:
```suggestion
    c->buf_usable_size = zmalloc_usable_size(c)-offsetof(client, buf);
```
I think this has the potential to capture any padding space after `buf` and might be clearer to read.

Reply:
ok, thanks